Header header
uint8 pump_mode
uint8 valve_1_mode
uint8 valve_2_mode
